package.json

```diff
@@ -36,64 +35,53 @@
     "launchdarkly-js-client-sdk": "^2.24.2",
     "lodash": "^4.17.21",
     "lodash.omit": "^4.5.0",
-    "pdfjs-dist": "3.4.120",
+    "pdfjs-dist": "3.9.179",
     "pinia": "^2.0.35",
     "pinia-class": "^0.0.3",
     "provinces": "^1.11.0",
     "qrcode.vue": "1.7.0",
     "regenerator-runtime": "^0.13.11",
     "register-service-worker": "^1.7.2",
     "sbc-common-components": "3.0.8",
-    "vue": "2.7.10",
+    "vue": "2.7.14",
     "vue-affix": "^0.5.2",
     "vue-hotjar": "^1.4.0",
     "vue-router": "^3.6.5",
     "vue2-filters": "^0.14.0",
-    "vuelidate": "^0.7.7",
+    "vuelidate": "0.6.2",
     "vuetify": "^2.6.15",
     "vuex": "^3.6.2"
   },
   "devDependencies": {
-    "@babel/plugin-proposal-private-methods": "^7.18.6",
     "@pinia/testing": "^0.0.16",
-    "@types/jest": "^27.5.2",
     "@types/lodash": "^4.14.194",
-    "@types/pdfjs-dist": "^2.10.378",
```

Comment:
You didn't see any use in this, or you couldn't find a version compatible with the package above?

Reply:
No need for it as per latest package version:
![no need for pdfjist types](https://github.com/bcgov/business-filings-ui/assets/122301442/72b18203-1199-48a3-8486-29e3b45cf213)

Reply:
Excellent. Thanks!